### DISCUSSION TITLE
Support for Xbox One S controller

### DISCUSSION THREAD

**Discussion by ifcbayle** [ndeadly/MissionControl#225] [Q&A]
Hi guys, 

First of all, congrats on your amazing job bringing the opportunity of connecting different kinds of controllers to this Console.

I'm facing some problems trying to connect my Xbox One S controller to Switch.
The console recognizes it but some buttons aren't recognized (such as X, both analog buttons and RB) and some are recognized wrongly (LB = X, View button = L, Menu button = R) . Any ideas on how to make it work properly?

Thanks a lot in advance.

**Comment by ndeadly** [comment]
You need to update the controller firmware. See here for more info    https://github.com/ndeadly/MissionControl#frequently-asked-questions
